dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/DubboProtocol.java

```diff
@@ -364,6 +367,9 @@ private ProtocolServer createServer(URL url) {
             // enable heartbeat by default
             .addParameterIfAbsent(HEARTBEAT_KEY, String.valueOf(DEFAULT_HEARTBEAT))
             .addParameter(CODEC_KEY, DubboCodec.NAME)
+            //todo
+            // enable pu server by default, this config should be closed when merging
+            .addParameter(IS_PU_SERVER_KEY, Boolean.TRUE.toString())
```

Comment:
Why is it enabled by default for now?

Reply:
This line should be deleted finally. For now, this line is used to make all dubbo protocol test cases use pu server.